### DISCUSSION TITLE
Nested Modal

### DISCUSSION THREAD

**Discussion by ELF-VIGO** [heroui-inc/heroui#2161] [Help]
I'd like to know how to use a Modal control inside another Modal. Is this posible? I'd tried in a sandbox but I couldn't open the nested Modal from the first one.

**Comment by ELF-VIGO** [comment]
I get to do that. Thank you.

**Reply by thedrivay** [reply]
How you've done that? Could you please help me by some demo syntax? It would really appreciated.

**Reply by ELF-VIGO** [reply]
You simply must use "useState variables" for every modal and it works fine. I'll try to put an example.

**Reply by thedrivay** [reply]
Thanks buddy, your advice really helped me.
One more thing, Have you been through developing carousel using NextUI?

**Reply by ELF-VIGO** [reply]
I'm using NextUI in a web project.

**Reply by tarekxsellami** [reply]
can you please show me how to do that? I am facing the same problem

**Reply by sujayxaradhya** [reply]
> can you please show me how to do that? I am facing the same problem

Hey, please share your code below. So I might be able to help you.
